Packs/CircleCI/Integrations/CircleCI/CircleCI.py

```diff
@@ -36,11 +36,19 @@ def get_workflows_list(self, vc_type: str, organization: str, project: str, page
 
     def get_last_workflow_runs(self, vc_type: str, organization: str, project: str, workflow_name: str,
                                page_token: Optional[str] = None):
+        branch: str = demisto.args().get('branch', '')
```

Comment:
it should be passed from the `circleci_workflow_last_runs_command` function

Reply:
done